sw/sw.mk

```diff
@@ -20,7 +20,7 @@ car-sw-all: car-sw-libs car-sw-tests
 # Libraries #
 #############
 
-CAR_SW_INCLUDES    = -I$(CAR_SW_DIR)/include -I$(CHS_SW_DIR)/include $(CHS_SW_DEPS_INCS)
+CAR_SW_INCLUDES    = -I$(CAR_SW_DIR)/include -I$(CAR_SW_DIR)/tests/bare-metal/safed -I$(CHS_SW_DIR)/include $(CHS_SW_DEPS_INCS)
```

Comment:
Why do we add a `bare-metal` folder? Everything here is already bare-metal.

Reply:
We can discuss this integration! I was taking as a referece the [aottaviano/offload-1](https://github.com/pulp-platform/carfield/tree/aottaviano/offload-1) branch, but still my branch is terribly messy :)